### ISSUE TITLE
Open VPN support?

### ISSUE THREAD

**Issue by artifactis** [RalfJung/noFUDroid#1]
Module did not remove the text from my 6.1 rooted droid turbo

**Comment by RalfJung** [comment]
Sorry, I need some more details here. What does this have to do with OpenVPN? Which text?

Right now, the module removes the warning about TLS certificates being installed. I am using that successfully on my Android 6 device.

**Comment by PwnicornDev** [comment]
See screenshot, this permanent warning is shown while connected to VPN. Not sure if it's within the scope of this module to address that as well.
![screenshot_2016-05-28-03-30-16](https://cloud.githubusercontent.com/assets/3987812/15624306/d22aa8e2-2484-11e6-8b5e-ff1ed2bd5148.png)

**Comment by artifactis** [comment]
Yes the screenshot describes my issue perfectly. When using my VPN the message is bit hidden. Would this be resolvable?

**Comment by RalfJung** [comment]
Ah, I see. Yeah I get that too. I guess this would be in scope, it could be considered FUD -- I would prefer the text to be something like "Your network traffic is routed through a VPN".

I will mark this as a feature request. I won't work on it myself anytime soon (too busy, sorry), but would be happy to review and test a pull request :)

**Comment by artifactis** [comment]
Thank you Ralf Jung

**Comment by artifactis** [comment]
Will be happy to test for you

**Comment by RalfJung** [comment]
As I said, I am not working on this. The bug is up for grabs for anyone who wants to work on it :)

**Comment by artifactis** [comment]
"Anytime soon" but okay...

**Comment by artifactis** [comment]
Anyway to hide the key at least?
